platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/LinkedEventIntakeWiring.java

```diff
@@ -59,8 +64,16 @@ public static LinkedEventIntakeWiring create(@NonNull final TaskScheduler<List<C
                 taskScheduler.buildInputWire("pause"),
                 consensusRoundOutput,
                 consensusRoundOutput.buildTransformer(
-                        "getMinimumGenerationNonAncient",
-                        consensusRound -> consensusRound.getGenerations().getMinGenerationNonAncient()),
+                        "getNonAncientEventWindow",
+                        consensusRound -> new NonAncientEventWindow(
```

Comment:
I think its better if we store the NonAncientEventWindow inside the round

Reply:
that might have some overlap with generations, so we might have to merge the two eventually

Reply:
I have moved the creation of the window into the call to the constructor for `ConsensusRound` and replaced this code with a get from the round.

Reply:
ok

Reply:
https://swirldslabs.slack.com/archives/C05M18GUMV1/p1703587747035269?thread_ts=1703164109.170669&cid=C05M18GUMV1

Reply:
code has been updated.